app/assets/stylesheets/custom.scss

```diff
@@ -175,3 +175,17 @@ input {
     color: $state-danger-text;
   }
 }
+
+.checkbox {
+  margin-top: -10px;
+  margin-bottom: 10px;
+  span {
+    margin-left: 20px;
+    font-weight: normal;
+  }
+}
+
+#session_remember_me {
```

Comment:
dùng class thay vì id để set css nhé

Reply:
cái id ni là do cái dòng phía trên hắn tự hắn thêm id vô trong cái tag html
<%= f.check_box :remember_me %>
mà ví dụ em giả đò ghi là <%= f.check_box :remember_me, class = "abc" %>
rồi đổi cái #session_remember_me thành .abc thì hắn chạy tầm bậy mất, chị có gợi ý be bé nào cho em fix nó với được ko ạ :dancing_men:

Reply:
https://stackoverflow.com/questions/30373569/add-class-with-check-box-in-rails-view